packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/avatar.tests.js

```diff
@@ -153,26 +190,20 @@ QUnit.module('ChatAvatar', moduleConfig, () => {
                 url: 'url',
             });
 
-            const $img = this.$element.find(`.${AVATAR_IMAGE_CLASS}`);
-
-            assert.strictEqual($img.attr('alt'), 'Avatar');
+            assert.strictEqual(this.getImage().attr('alt'), 'Avatar');
         });
 
         QUnit.test('img element should have correct src attribute', function(assert) {
             this.reinit({ url: 'url' });
 
-            const $img = this.$element.find(`.${AVATAR_IMAGE_CLASS}`);
-
-            assert.strictEqual($img.attr('src'), 'url');
+            assert.strictEqual(this.getImage().attr('src'), 'url');
         });
 
         QUnit.test('img element should have correct src attribute if url was changed in runtime', function(assert) {
             this.reinit({ url: 'url' });
             this.instance.option({ url: 'New url' });
 
-            const $img = this.$element.find(`.${AVATAR_IMAGE_CLASS}`);
-
-            assert.strictEqual($img.attr('src'), 'New url');
+            assert.strictEqual(this.getImage().attr('src'), 'New url');
         });
     });
 });
```

Comment:
[test coverage]
In the `messageGroup.tests.js` u can find a module with tests checking that all necessary properties are correctly passed to Avatar.
Please add a test for this new option

Reply:
[1d45201](https://github.com/DevExpress/DevExtreme/pull/28237/commits/1d4520125b38d3f2268f5c9ced75c733b486c1ce)